Source/WTF/wtf/PlatformUse.h

```diff
@@ -157,6 +157,10 @@
 #define USE_EXECUTE_ONLY_JIT_WRITE_FUNCTION 1
 #endif
 
+#if PLATFORM(VISION)
+#define USE_STATIC_IPAD_USER_AGENT_VALUE 1
```

Comment:
I know this predates you, but this is an ENABLE, NOT a USE ("use a particular third-party library or optional OS service")

Reply:
Oh, I could move it to the PlatformEnableCocoa header, and if it's only used in one or two places maybe change it to ENABLE_STATIC_IPAD_USER_AGENT_VALUE.